qiskit/providers/ibmq/jobmanager.py

```diff
@@ -0,0 +1,332 @@
+            experiments = transpile(circuits=experiments, backend=backend)
+        experiment_list = self._split_experiments(experiments, backend=backend,
+                                                  max_experiments_per_job=max_experiments_per_job)
+
+        qobjs = []
+        for experiment in experiment_list:
+            qobjs.append(assemble(experiment, backend=backend, shots=shots))
+
+        self._future = self._executor.submit(self._submit_async, qobjs=qobjs,
+                                             backend=backend, name_prefix=name_prefix)
+        return len(qobjs)
+
+    def _submit_async(
+            self,
+            qobjs: List[Qobj],
+            backend: IBMQBackend,
+            name_prefix: Optional[str] = None
+    ) -> None:
+        """Submit jobs to IBM Q asynchronously.
+
+        Args:
+            qobjs: A list of qobj's to run.
+            backend: Backend to execute the experiments on.
+            name_prefix: Prefix of the job name.
+        """
+        name_prefix = name_prefix or datetime.utcnow().isoformat()
+        start_index = 0
+        end_index = 0
+        try:
+            for i, qobj in enumerate(qobjs):
+                job = backend.run(qobj=qobj, job_name="{}_{}".format(name_prefix, i))
+                end_index = start_index + len(qobj.experiments) - 1
+                setattr(job, 'start_index', start_index)
+                setattr(job, 'end_index', end_index)
+                start_index = end_index + 1
+                self._jobs.append(job)
+        except Exception as err:  # pylint: disable=broad-except
+            self._future_captured_exception = err  # type: ignore[assignment]
+            err_msg = "Unable to submit job {} for experiments {}-{}.".format(
+                i, start_index, end_index)
+            self._future_error_msg = err_msg  # type: ignore[assignment]
+
+            # Cancel all previously submitted jobs
+            try:
+                for job in self._jobs:
+                    job.cancel()
+            except Exception:  # pylint: disable=broad-except
+                pass
+
+    @requires_submit
+    def jobs(self) -> List[IBMQJob]:
+        """Return a list of submitted jobs.
+
+        Returns:
+            A list of IBMQJob instances that represents the submitted jobs.
+        """
+        return self._jobs
+
+    @requires_submit
+    def status(self) -> List[Union[JobStatus, None]]:
+        """Return the status of each job.
+
+        Returns:
+            A list of job statuses. The entry is ``None`` if the job status
+                cannot be retrieved due to server error.
+        """
+        statuses = []
+        for i, job in enumerate(self._jobs):
+            try:
+                statuses.append(job.status())
+            except JobError as err:
+                logger.warning("Unable to retrieve status for job %d (job ID=%s): %s",
+                               i, job.job_id(), str(err))
+                statuses.append(None)
```

Comment:
Would this cause some ripples in `report()`? It seems it would not account for the job in the non-detailed output (which can be considered "by design"), but more importantly can you check it does not cause an error in the function (ie. when accessing `status.value` or similar cases)?

Reply:
Not counting in the summary report is by design; fixed `status.value`. We really should have a `JobStatus.UNKNOWN`.